src/repository/screens/read-me.screen.js

```diff
@@ -1,46 +1,20 @@
 import React, { Component } from 'react';
-import { WebView, StyleSheet, View, Text } from 'react-native';
+import { WebView } from 'react-native';
 import { connect } from 'react-redux';
 
 import { ViewContainer, LoadingContainer } from 'components';
-import { normalize } from 'config';
-import { getReadMe } from '../repository.action';
```

Comment:
Thanks a lot for the PR.

Code seems okay, but I'm a bit worried about downloading the whole README file contents on the repository screen. If the user doesn't consult the README, it's a big waste of data for nothing.

Could you instead move back README download in the ReadMe screen, and just test for its existence in the Repository screen? It's achievable using a HEAD request instead of GET:

Repo with README.md responds with response code 200:

🍺  ~/git-point (markdown-renderer)*$ curl --head https://api.github.com/repos/gitpoint/git-point/readme
HTTP/1.1 200 OK
Repo without it responds with 404 Not found:

🍺  ~/git-point (markdown-renderer)*$ curl --head https://api.github.com/repos/machour/git-point-playground/readme
HTTP/1.1 404 Not Found
Thanks in advance!

Reply:
@machour Great suggestion! But I want to wait @housseindjirdeh to confirm the purpose of `JSON.parse(readMe).message`. If it can be skipped, I will update codes as you say.

Reply:
If I'm not mistaken, I just included it since the prop was a JSON string. Please feel free to remove it entirely if the object returned is fine (would rather prefer we don't have it to be honest :))

And cheers, appreciate you asking :raised_hands:

Reply:
@machour @housseindjirdeh I rewrite my codes and introduce a new method called `fetchUrlHead` in `src/api/index.js`.